src/Sulu/Bundle/AdminBundle/Gruntfile.js

```diff
@@ -121,25 +137,44 @@ module.exports = function(grunt) {
                 files: [
                     {src: ['Resources/public/dist/app.min.js'], dest: 'Resources/public/dist/app.min.js'}
                 ]
+            },
+            buildLogin: {
+                options: {
+                    variables: {
+                        '/js/login.js': '/dist/login.js'
+                    },
+                    prefix: ''
+                },
+                files: [
+                    {src: ['Resources/public/dist/login.min.js'], dest: 'Resources/public/dist/login.min.js'}
+                ]
             }
         },
         requirejs: {
-            compile: {
+            app: {
                 options: {
                     baseUrl: 'Resources/public/js/',
                     mainConfigFile: 'Resources/public/js/main.js',
                     preserveLicenseComments: false
                 }
+            },
+            login: {
```

Comment:
Is there a reason you have an own js file for the login now? Didn't we say we should load the entire one, because of the http caching?